package.json

```diff
@@ -14,16 +14,21 @@
   },
   "dependencies": {
     "babel-core": "^6.26.3",
-    "babel-loader": "^8.1.0",
     "babel-polyfill": "^6.26.0",
     "babel-preset-env": "^1.7.0",
     "babelify": "^10.0.0",
     "eslint-plugin-react": "^7.20.3",
-    "webpack": "^4.43.0",
+    "react": "^16.13.1",
```

Comment:
I suspect we might want to use the same version of React as https://studio.code.org does, particularly for when we are running as hosted content.  I notice that our Oceans repo does [that](https://github.com/code-dot-org/ml-activities/blob/main/package.json#L38).